File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/ValidationResponse.java
package com.lattis.ellipse.data.network.model.response;

import com.google.gson.annotations.SerializedName;

public class ValidationResponse extends AbstractApiResponse {

    @SerializedName("payload")
    private UserResponse userResponse;

    public UserResponse getUserResponse() {
        return userResponse;
    }

    @Override
    public String toString() {
        return "ValidationResponse{" +
                "userResponse=" + userResponse +
                '}';
    }
}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/authentication/ValidationResponse.kt
package com.lattis.data.entity.response.authentication

import com.google.gson.annotations.SerializedName
import com.lattis.data.entity.response.user.UserResponse
import com.lattis.data.entity.response.base.AbstractApiResponse

class ValidationResponse : AbstractApiResponse() {
    @SerializedName("payload")
    private var userResponse: UserResponse? = null
    fun getUserResponse(): UserResponse? {
        return userResponse
    }

    override fun toString(): String {
        return "ValidationResponse{" +
                "userResponse=" + userResponse +
                '}'
    }
}